### ISSUE TITLE
Question regarding effect of threshold criteria on MCI subgroup classification

### ISSUE THREAD

**Issue by rushin682** [rlckd/MCI-subtype#1]
Hello,

Thanks for the great paper! 
I noticed that you and your team had performed a comparative analysis of alterations in the threshold for MCI subgroup classification (Figure S3 b-j). I am curious to what does this threshold represent? Are these the logits from the sigmoid activation in the final layer? If so, do you interpret them as confidence with which the MCI are classified as MCI-AD or MCI-CN? 

Could you please help me understand the importance of the threshold analysis? I intend to use your conceptual approach for an independent histopathology study.

Thank you for your time in advance. 😄
